src/app/loadout-builder/process.ts

```diff
@@ -473,10 +457,7 @@ function generateMixesFromPerksOrStats(
     return [];
   }
 
-  const statsByHash = _.keyBy(stats, (stat) => stat.statHash);
-  const mixes: number[][] = [
-    getBaseStatValues(statsByHash, item, assumeArmor2IsMasterwork, lockedModStats),
-  ];
```

Comment:
This is the block where the statsByHash was pulled from. I just moved it into that function as I use it in two spots now.